app/_src/networking/dns.md

```diff
@@ -119,8 +119,9 @@ Consuming a service handled by {{site.mesh_product_name}} DNS, whether from {{si
 <kuma-enabled-pod>$ curl http://echo-server_echo-example_svc_1010.mesh:80
 <kuma-enabled-pod>$ curl http://echo-server_echo-example_svc_1010.mesh
 ```
+Be aware of DNS standards ([RFC1035](https://www.ietf.org/rfc/rfc1035.txt)) if using underscores in the service name.
 
-A DNS standards compliant name is also available, where the underscores in the service name are replaced with dots. For example:
+A DNS standards-compliant name is also available, where the underscores in the service name are replaced with dots. For example:
```

Comment:
* I removed the word "standards-compliant" because it is redundant. The word "DNS" already implies that the name complies with the DNS standards.
* I added the word "can" before "be used" to make it clear that this is just one option.
* I added the word "the" before "service name" for consistency.
```suggestion
A DNS-compliant ([RFC 1035](https://www.ietf.org/rfc/rfc1035.txt) name can also be used, where the underscores in the service name are replaced with dots. For example:
```